api/api-service.js

```diff
@@ -9,20 +9,19 @@ var Seneca = require('seneca')
 
 var server = new Hapi.Server()
 
-server.connection({ 
+server.connection({
   port: PORT
 })
 
 server.register({
-  register: Chairo, 
+  register: Chairo,
   options:{
     seneca: Seneca({
       tag: 'api',
       log: 'silent',
-      legacy: { logging: false },
-      internal: { logger: require('seneca-demo-logger') },
       debug: {short_logs:true}
     })
+    .use('demo-logger')
```

Comment:
@rjrodger exactly.
To recap, I need to:
- revert to the special way to register the logger
- remove log:'silent'

Is that correct?